CPT331.iOS/CPT331.iOS/EventCategory.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  EventCategory.swift
+//  CPT331.iOS
+//
+//  Created by Peter Weller on 8/10/2016.
+//  Copyright © 2016 Peter Weller. All rights reserved.
+//
+
+import UIKit
+
+enum EventCategory {
```

Comment:
Are the event categories hard coded to these options?

I think this data type as an enum make it difficult to add new categories, particularly if they are database driven.